From 0dfe157012a8f9a2031d015e1d27fceca0467f43 Mon Sep 17 00:00:00 2001
From: MarijnS95 <marijns95@gmail.com>
Date: Mon, 9 Sep 2019 14:18:43 +0200
Subject: [PATCH 1/1] [Q] Never build bootctrl the old way.

We are moving all our platforms to build bootctrl using the BluePrint
mechanism. Remove the makefile alltogether to prevent building bootctrl
on unsupported platforms (msm8998 for example).

Change-Id: I999da77b7bba9aca4a2f17045b6eff010aa5faf1
Signed-off-by: MarijnS95 <marijns95@gmail.com>
---
 Android.mk | 35 -----------------------------------
 1 file changed, 35 deletions(-)
 delete mode 100644 Android.mk

diff --git a/Android.mk b/Android.mk
deleted file mode 100644
index c5a9b16..0000000
--- a/Android.mk
+++ /dev/null
@@ -1,35 +0,0 @@
-# Preset TARGET_USES_HARDWARE_QCOM_BOOTCTRL for existing platforms.
-ifneq ($(filter msm8996 msm8998 sdm710,$(TARGET_BOARD_PLATFORM)),)
-TARGET_USES_HARDWARE_QCOM_BOOTCTRL := true
-endif
-
-ifeq ($(strip $(TARGET_USES_HARDWARE_QCOM_BOOTCTRL)),true)
-# TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
-ifneq ($(BOARD_IS_AUTOMOTIVE),true)
-LOCAL_PATH := $(call my-dir)
-
-# HAL Shared library for the target. Used by libhardware.
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/gpt-utils/inc
-LOCAL_CFLAGS += -Wall -Werror
-LOCAL_SHARED_LIBRARIES += liblog libgptutils libcutils
-LOCAL_HEADER_LIBRARIES := libhardware_headers libsystem_headers
-LOCAL_SRC_FILES := boot_control.cpp
-LOCAL_MODULE_RELATIVE_PATH := hw
-LOCAL_MODULE := bootctrl.$(TARGET_BOARD_PLATFORM)
-LOCAL_MODULE_OWNER := qcom
-LOCAL_PROPRIETARY_MODULE := true
-include $(BUILD_SHARED_LIBRARY)
-
-# Static library for the target. Used by update_engine_sideload from recovery.
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/gpt-utils/inc
-LOCAL_CFLAGS += -Wall -Werror
-LOCAL_SHARED_LIBRARIES += liblog libgptutils libcutils
-LOCAL_HEADER_LIBRARIES := libhardware_headers libsystem_headers
-LOCAL_SRC_FILES := boot_control.cpp
-LOCAL_MODULE := bootctrl.$(TARGET_BOARD_PLATFORM)
-include $(BUILD_STATIC_LIBRARY)
-
-endif
-endif
-- 
2.23.0

